src/LogFileReader.cpp

```diff
@@ -29,9 +29,10 @@
 #include <QFile>
 #include <QString>
 #include <QTextStream>
-
 #include "LogFileReader.h"
-
+#include <iostream>
+#include <QTime>
+using namespace std;
```

Comment:
This line is also not necessary, but I'd also like to note that the directive `using namespace ...;` in general is frowned upon because it creates ambiguity for where some words come from.